docs/en/user-guide/How-To/index.rst

```diff
@@ -119,39 +190,47 @@ Note:
 
 Action:
 
-1. Select the wastewater structure with the i-button
+1. Select in layer vw_tww_wastewater_structure the wastewater structure with the i-button
 
-2. Select the wastewater nodes tab in the Feature Attributes window
+2. Select the **Wastewater Nodes** tab in the Feature Attributes window
 
-3. Select a hydr_geometry in the fk_hydr_geometry field or use the + button to create a new hydr_geometry.
+or direct way: Select in layer vw_wastewater_node the node with the i-button
```

Comment:
or in a more direct way: Select the node with the i-button in the layer vw_wastewater_node